scripts/attempt-backport.js

```diff
@@ -27,9 +32,9 @@ module.exports = function (app) {
     const options = { owner, repo, prId, logger: event.logger }
 
     debug(`/${owner}/${repo}/pull/${prId} sync`)
-    queueAttemptBackport(options, 7, false)
-    queueAttemptBackport(options, 6, true)
-    queueAttemptBackport(options, 4, true)
+    nodeVersions.forEach(function(node) {
```

Comment:
Oh thanks for elaborating. I'm not aware of the benefits using `for ... of`, got any pointers / resources for me to read up on?